src/compositions/FieldsetHeader/FieldsetHeader.tsx

```diff
@@ -60,7 +61,7 @@ const renderType = (type: Type, id: string, size: Size, active: boolean, disable
     return null;
 };
 
-export const FieldsetHeader = ({
+export function FieldsetHeader({
```

Comment:
Any reason?

Reply:
nope will revert and refactor to use `FC`

Reply:
cd99033